src/components/CheckboxGroup/CheckboxGroup.spec.tsx

```diff
@@ -0,0 +1,68 @@
+import * as React from 'react';
+import { createUniDriverFactory } from 'wix-ui-test-utils/uni-driver-factory';
+import { isUniEnzymeTestkitExists } from 'wix-ui-test-utils/enzyme';
+import { isUniTestkitExists } from 'wix-ui-test-utils/vanilla';
+import { mount } from 'enzyme';
+import { checkboxGroupDriverFactory } from './CheckboxGroup.driver';
+import { CheckboxGroup } from './';
+import { Checkbox } from '../Checkbox';
+import { checkboxGroupTestkitFactory } from '../../testkit';
+import { checkboxGroupTestkitFactory as enzymeCheckboxGroupTestkitFactory } from '../../testkit/enzyme';
+
+const el = (
+  <CheckboxGroup>
+    <Checkbox
+      name="group1"
+      onChange={val => console.log(val)}
```

Comment:
I think it's best not to use `console.log` so that it won't add "noisy" logs when running tests and such (if we add click tests for instance)